be2-scala/src/main/scala/ch/epfl/pop/pubsub/graph/validators/ElectionValidator.scala

```diff
@@ -101,6 +104,18 @@ sealed class ElectionValidator(dbActorRef: => AskableActorRef) extends MessageDa
     }
   }
 
+  private def allVoteHaveValidIndex(votes: List[VoteElection], q2ballots: Map[Hash, List[String]]) =
+    votes.forall(
+      voteElection => {
+        val vote = voteElection.vote match {
+          case Some(v :: _) => v
+          case _ => -1
+        }
+        // check if the ballot is available
+        vote < q2ballots(voteElection.question).size
+      }
+    )
```

Comment:
```suggestion
    votes.forall { voteElection =>
        val vote = voteElection.vote match {
          case Some(v :: _) => v
          case _ => -1
        }
        // check if the ballot is available
        vote < q2ballots(voteElection.question).size
      }
```
Usually, this is the more conventional lambda syntax.